.github/workflows/code-quality.yml

```diff
@@ -9,16 +9,14 @@ on:
   workflow_dispatch:
     inputs:
       dbt_adapters_branch:
-          description: "The branch of dbt-adapters to evaluate"
-          type: string
-          required: true
-          default: "main"
+        description: "The branch of dbt-adapters to evaluate"
+        type: string
+        default: "main"
   workflow_call:
     inputs:
       dbt_adapters_branch:
         description: "The branch of dbt-adapters to evaluate"
         type: string
-        required: true
```

Comment:
What happens when someone puts empty string? Isn't this here to protect against that user error?

Reply:
I think if you pass the argument in the `with` clause, it will pass the required check, even if an empty string is passed. If we want to keep `required: true`, then we should eliminate the `default: "main"` as it will never get used.

Reply:
>  we should eliminate the default: "main" as it will never get used
I don't agree with this. It will default to main in text in the box, no?
> I think if you pass the argument in the with clause, it will pass the required check, even if an empty string is passed.
But this is a great point ^^

Reply:
Which way is preferred? Should this be a required parameter or a defaulted one?